### PR TITLE
Add deskocat agent and a skill to assign issues to Copilot

### PR DESCRIPTION
**Pull request by tidy-dev**
## Description
**New agent and assignment workflow documentation:**

* Added `.github/agents/deskocat.agent.md` to define the responsibilities, workflow, risk classification, architecture, and testing guidelines for the `deskocat` agent, providing a detailed reference for contributors and reviewers.

**Copilot assignment skill and tooling:**

* Added `.github/skills/assign-copilot/SKILL.md` to document the process for assigning issues to Copilot, including usage instructions, requirements, and available custom agents.
* Added `.github/skills/assign-copilot/assign.sh`, a Bash script that automates assigning a GitHub issue to Copilot or a specified custom agent, handling repository detection and assignment via the GitHub CLI and API.

## Release notes

Notes: no-notes

### CODE REVIEW

File: .github/agents/deskocat.agent.md [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+**Edge cases considered**: [list them]
+
+## Test Plan
+
+**Automated**: [tests added or updated]
+**Manual QA**: [steps for reviewer to verify]
+
+## Screenshots
+
+[If UI changes, include before/after screenshots]
```
Comment (niik) [inline .github/agents/deskocat.agent.md:105]: We don't have any tools for it to be able to do this right now though right? Does it still make sense to include this?

File: .github/agents/deskocat.agent.md [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+```
+
+---
+
+## Risk Classification
+
+Classify every change by the highest-risk area it touches.
+
+### Critical — Auto-update & Installation
+Changes here can **trap users on a broken version** with no way to update. Require extensive manual QA on both macOS and Windows.
```
Comment (niik) [inline .github/agents/deskocat.agent.md:124]: :heart:

**Review by niik — APPROVED** [review]
I think this is a great start, well written and includes a lot of Desktop-isms, nice

File: .github/agents/deskocat.agent.md (hunk 1 above)
Comment (tidy-dev) [reply .github/agents/deskocat.agent.md:105]: Eh.. I think it is ok to have in place... hoping playwright can help us here maybe.... it doesn't seem to hurt when using it.
